packages/crypto/src/primitives/xchacha20-poly1305.ts

```diff
@@ -195,17 +193,15 @@ export class XChaCha20Poly1305 {
     data: Uint8Array;
     key: Jwk;
     nonce: Uint8Array;
-  }): Promise<{ ciphertext: Uint8Array, tag: Uint8Array }> {
+  }): Promise<Uint8Array> {
+  // }): Promise<{ ciphertext: Uint8Array, tag: Uint8Array }> {
```

Comment:
```suggestion

```

Reply:
Good catch.  Thanks!

Addressed in [38ba056](https://github.com/TBD54566975/web5-js/pull/393/commits/38ba056ca3bfc9486f8e1045d245f66f98f31fc0)